boefjes/boefjes/job_handler.py

```diff
@@ -104,7 +105,11 @@ def handle(self, boefje_meta: BoefjeMeta) -> None:
                     reference, valid_time=datetime.now(timezone.utc)
                 )
             except ObjectNotFoundException as e:
-                raise ObjectNotFoundException(f"Object {reference} not found in Octopoes") from e
+                if reference.class_type == KATFindingType:
+                    logger.exception("Object {%s} not found in Octopoes", e)
+                    return
+                else:
+                    raise ObjectNotFoundException(f"Object {reference} not found in Octopoes") from e
```

Comment:
```suggestion
                logger.info("Can't run boefje because OOI does not exist anymore", boefje_id=boefje_meta.boefje.id, ooi=reference, task_id=boefje_meta.id)
                return
```

The problem is not specific for KATFindingType, a boefje for every object type can be scheduled and then be deleted before the runner can fetch the input OOI. So I don't see why we should special case `KATFindingType`.

We also shouldn't use logger.exception because that would still log the exception just like we did before. What needs to be changed is that we should not log an exception because nothing actually goes wrong.

Reply:
The idea here is to not crash on objects that we "expect" to have been deleted; these are typically objects like KATFindingTypes whom are being affirmed while being deleted. As we have no access to the origin that is being to be yielded we can deduce from the object type that in this case deletion is expected. For other case we might want to crash because we have found abnormal behavior?

Reply:
The race condition can happen for every object. Every object can be deleted and if the object can be used as input OOI, it is possible that the deletion happens at the same moment the scheduler sends the OOI as input to the boefje runner.

That we see it more often and/or can reproduce it with KATFindingType doesn't really matter, it's just nature of race conditions that you don't always see them or can reproduce them.

Reply:
Right, but this does not solve the question on whether you would like to crash in certain cases.... In any case, I'll yield to your insight on this matter.